packages/orbit-components/src/CarrierLogo/CarrierLogo.stories.tsx

```diff
@@ -1,162 +1,104 @@
 import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import { SIZE_OPTIONS, CARRIER_TYPE_OPTIONS } from "./consts";
 import RenderInRtl from "../utils/rtl/RenderInRtl";
 
 import CarrierLogo from ".";
 
-export default {
+const meta: Meta<typeof CarrierLogo> = {
   title: "CarrierLogo",
-};
-
-export const OneCarrier = ({ size, dataTest, carriers }) => {
-  return <CarrierLogo size={size} carriers={carriers} dataTest={dataTest} />;
-};
-
-OneCarrier.story = {
-  name: "One carrier",
+  component: CarrierLogo,
 
   parameters: {
     info: "Carrier logo displays one or more logos of transport carriers. It is usually together with the travel itinerary. Visit Orbit.Kiwi for more detailed guidelines.",
   },
-};
 
-OneCarrier.args = {
-  size: SIZE_OPTIONS.LARGE,
-  dataTest: "test",
-  carriers: [{ code: "FR", name: "Ryanair" }],
-};
-
-OneCarrier.argTypes = {
-  size: {
-    options: Object.values(SIZE_OPTIONS),
-    control: { type: "select" },
+  args: {
```

Comment:
Cannot make them `rounded`?

Reply:
Good point, thx. It should be fixed now. ✅